src/App.tsx

```diff
@@ -15,71 +16,84 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface AppState {
+  selectedGood: string;
+}
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, AppState> {
+  state: AppState = {
+    selectedGood: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+  handleSelectGood = (good: string) => {
+    this.setState({ selectedGood: good });
+  };
+
+  handleClearSelection = () => {
+    this.setState({ selectedGood: '' });
+  };
+
+  render() {
+    const { selectedGood } = this.state;
+    const title = selectedGood ? `${selectedGood} is selected` : 'No goods selected';
+    const buttonClear = selectedGood && (
```

Comment:
You don't re-use it, so I don't see any reason to move this part of `JSX` to a variable
Just write it in place